src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,69 @@
+package duke.task;
+
+/**
+ * Represent the task for the Duke program.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    /**
+     * Return the symbol of the status of this task.
+     * @return ✓ if task is done, ✘ if task is not done
```

Comment:
I quite like the clarity in your comments, some quick tips for readability and adherence to javadocs standards - JavaDocs general comment section should have one line break from the remainder.

For instance, it should look like this:

* Return the symbol of the status of this task.
*
* @return ✓ if task is done, ✘ if task is not done